src/test/java/me/moodcat/database/DatabaseTestModule.java

```diff
@@ -17,6 +20,7 @@
     @Override
     protected void configure() {
         super.configure();
+        bind(RoomDAO.class).to(H2RoomDAO.class);
```

Comment:
You did this as well on creation on App, is this duplicate or does it serve another purpose?